src/overlays/actors/ovl_En_Ruppecrow/z_en_ruppecrow.h

```diff
@@ -3,15 +3,44 @@
 
 #include "global.h"
 
+#define ENRUPPECROW_GET_PATH(thisx) (((thisx)->params & 0xFC00) >> 0xA)
+
+#define ENRUPPECROW_RUPEE_COUNT 20
+#define ENRUPPECROW_LIMB_COUNT 9
+#define ENRUPPECROW_LIMB_POS_COUNT 4
+
+enum {
+    ENRUPPECROW_EFFECT_NONE = 0,
+    ENRUPPECROW_EFFECT_ICE = 10,
+    ENRUPPECROW_EFFECT_LIGHT = 20,
+};
+
 struct EnRuppecrow;
 
 typedef void (*EnRuppecrowActionFunc)(struct EnRuppecrow*, GlobalContext*);
 
 typedef struct EnRuppecrow {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x94];
-    /* 0x01D8 */ EnRuppecrowActionFunc actionFunc;
-    /* 0x01DC */ char unk_1DC[0x128];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ EnItem00* rupees[ENRUPPECROW_RUPEE_COUNT];
+    /* 0x194 */ SkelAnime skelAnime;
+    /* 0x1D8 */ EnRuppecrowActionFunc actionFunc;
+    /* 0x1DC */ UNK_TYPE4 unk_1DC; // unused
+    /* 0x1E0 */ Vec3s joinTable[ENRUPPECROW_LIMB_COUNT];
+    /* 0x216 */ Vec3s morphTable[ENRUPPECROW_LIMB_COUNT];
+    /* 0x24C */ Path* path;
+    /* 0x250 */ s32 currentPoint;
+    /* 0x254 */ ColliderJntSph collider;
+    /* 0x274 */ ColliderJntSphElement colliderElement;
+    /* 0x2B4 */ u16 isGoingCounterClockwise;
+    /* 0x2B8 */ f32 speedModifier;
+    /* 0x2BC */ s16 rupeeIndex;
+    /* 0x2BE */ s16 yOffset;
+    /* 0x2C0 */ UNK_TYPE4 unk_2C0; // unused
+    /* 0x2C4 */ u8 currentEffect; 
+    /* 0x2C8 */ f32 unk_2C8; // set but not used
+    /* 0x2CC */ f32 unk_2CC; // set but not used
+    /* 0x2D0 */ f32 unk_2D0;
```

Comment:
This looks like a timer for playing a sound effect. Could be the time it takes ice to fall.

Reply:
Implemented `unk_2D0` as `iceSfxTimer`, it is used as a timer for the "ice freeze" sound effect when struck by an ice arrow.